File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.swift
// Swift syntax highlighting test
import Foundation

func greet(_ name: String) -> String {
    return "Hello, \(name)!"
}

struct Config {
    let version: String
    var enabled: Bool = true
    let count: Int
}

let message = greet("World")
print(message)

let items = [1, 2, 3, 4, 5]
for item in items {
    print("Item: \(item)")
}

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.nim
# Nim syntax highlighting test
import strformat

proc greet(name: string): string =
  fmt"Hello, {name}!"

type
  Config = object
    version: string
    enabled: bool
    count: int

proc main() =
  let message = greet("World")
  echo message

  let items = @[1, 2, 3, 4, 5]
  for item in items:
    echo fmt"Item: {item}"

when isMainModule:
  main()
